### ISSUE TITLE
More minigames

### ISSUE THREAD

**Issue by def-** [headshot2017/teeware-mod#21]
A few people told me that there should be more minigames.

**Comment by headshot2017** [comment]
yep. currently developing a "piggyback" minigame. should be done later today. i will be developing perhaps 10 more minigames before v1.2
